opentelemetry/src/global/internal_logging.rs

```diff
@@ -4,6 +4,9 @@
 /// **internally within OpenTelemetry code** or for **custom exporters, processors and other plugins**. They are not designed
```

Comment:
Changes here cause the macros to print to stdout if you are running in a test. I find it super convenient to see internal logging when debugging tests.  You need to opt into this with `--nocapture` when you run them, otherwise the output goes to `/dev/null`.